src/Alarm.cpp

```diff
@@ -13,24 +21,36 @@
 #include "Logger.h"
 #include "ComponentTester.h"
 
+/**
+  Base contructor that writes that the Alarm Object has been created
+*/
 Alarm::Alarm(){
-  Logger::Log("Alarm has been created");
+  Serial.write(13100); // 13100 = Log, Debug, Alarm object successfully created
 }
 
-
+/**
+  Takes in the base calibration of the light level using the
+    determineBasePhotoresistorReading function and alerts of failed action
+      if alertFailedAction and return isCalibrated as false, else it returns
+      isCalibrated as true.
+    @param isCalibrated is set to true or false
+*/
 void Alarm::calibrate(){
   determineBasePhotoresistorReading();
   if(_photoR->takeReading() - _threshold < _baseReading){
     alertFailedAction();
     _isCalibrated = false;
   } else {
     alertSuccessfulAction();
-    Logger::Log("Alarm Successful Calibration");
+    Serial.write(30004); // 30004 = Command, N/A, Execute Command: Calibrate
```

Comment:
The 30004 command is the command I send from the PI to the Arduino when I want it to calibrate. This isn't the right 5 digit number.